support directory missing in 0.8.0

### Describe the Bug

The published npm package for 0.8.0 does not contain the support folder which means it fails on install as it can not compile the java libraries 


### Steps to Reproduce


1. Install version 0.8.0 using `npm install xsd-schema-validator@0.8.0`
2. The following shows
```
npm ERR! code 1
npm ERR! path /workspaces/electronic-prescription-service-api/node_modules/xsd-schema-validator/node_modules/xsd-schema-validator
npm ERR! command failed
npm ERR! command sh -c node ./lib/post-install.js
npm ERR! [xsd-schema-validator] Compiling helper...
npm ERR! Error: Failed to compile helper (exitCode=2)
npm ERR!     at ChildProcess.<anonymous> (/workspaces/electronic-prescription-service-api/node_modules/xsd-schema-validator/node_modules/xsd-schema-validator/lib/validator.js:57:13)
npm ERR!     at ChildProcess.emit (node:events:514:28)
npm ERR!     at ChildProcess._handle.onexit (node:internal/child_process:294:12)

npm ERR! A complete log of this run can be found in: /home/vscode/.npm/_logs/2024-01-16T08_26_22_686Z-debug-0.log
```


### Expected Behavior

The package should install


### Environment

- node version 20.10.0
- OS: ubuntu
- Library version: 0.8.0

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "which": "^4.0.0"
   },
   "files": [
-    "lib"
+    "lib",
+    "support"
   ]
 }

